### PR TITLE
Fixing HTML issue

### PR DESCRIPTION
**Pull request by NickCEBM**
Broken text at the top of page. Moving that link down.

### CODE REVIEW

**Comment by NickCEBM** [comment]
No idea why the checks failed though.

File: euctr/templates/about.html [hunk 1]
```diff
@@ -154,38 +155,39 @@ <h2 id="inconsistent-data">What does “Inconsistent Data” Mean for a Trial?</
         <p>Currently, we identify the following issues:
 
         <ol>
-          <li>A trial in which all sites are either “completed” or “terminated”
+          <li>A trial in which all sites are either “completed” or “terminated
```
Comment (madwort) [inline euctr/templates/about.html:158]: looks like a typo here - I think the `"` needs to be reinstated?

Comment (NickCEBM) [reply euctr/templates/about.html]: Always one...

**Review by madwort — APPROVED** [review]
looks good to me

**Comment by madwort** [comment]
Travis CI passes (don't worry about the other CI for now)

**Pull request merged**